<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addTraining</class>
 <widget class="QDialog" name="addTraining">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1536</width>
    <height>864</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1536</width>
     <height>864</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#frame {	
	color: rgb(255, 255, 255);
	background-image: url(:/bg/fitora.png);
    background-position: center;
    background-repeat: no-repeat;
    background-size: cover;
    border: none;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QCheckBox" name="day7">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>440</y>
      <width>93</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>fri</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="capacitylineEdit">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>330</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="coaches">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>390</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QTimeEdit" name="StimeEdit">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>390</y>
      <width>118</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="DtimlineEdit">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>390</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="day3">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>440</y>
      <width>93</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>mon</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>330</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>name </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>510</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="day5">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>440</y>
      <width>93</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>wed</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>330</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>capacity</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>440</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>days</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="day1">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>440</y>
      <width>93</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>sat</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="day4">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>440</y>
      <width>93</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>tue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeletepushButton">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>510</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QComboBox" name="namecomboBox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>330</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>390</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>coach </string>
    </property>
   </widget>
   <widget class="QCheckBox" name="day6">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>440</y>
      <width>93</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>thu</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="day2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>440</y>
      <width>93</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>sun</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>390</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>start time </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>390</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>duration time </string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
